component/site/models/checkout.php

```diff
@@ -1104,6 +1100,10 @@ public function orderplace()
             return false;
         }
 
+        if ($row->order_status == Redshop::getConfig()->get('CLICKATELL_ORDER_STATUS')) {
+            RedshopHelperClickatell::clickatellSMS($orderId);
```

Comment:
@hoang-tran-redweb From now on, please help to add \ before class name as best pratice.
Thanks.
H.A

Reply:
@anhredweb I fixed.